tst/files/human-created/testNonConjugacy.tst

```diff
@@ -11,11 +15,18 @@ gap> TestConjugacy := function(K, H, n)
 >     od;
 >     return true;
 > end;;
+
+#
 gap> K := AlternatingGroup(15);;
 gap> H := SymmetricGroup(25);;
 gap> TestConjugacy(K, H, 20);
 true
+
+#
 gap> K := AlternatingGroup(15);;
 gap> H := DirectProduct([SymmetricGroup(10),SymmetricGroup(10),SymmetricGroup(5)]);;
 gap> TestConjugacy(K, H, 20);
 true
+
+#
+gap> STOP_TEST("testNonConjugacy.tst", 1);
```

Comment:
Just FYI you can actually omit the `, 1`, it is only there for historical reasons (and most callers still have it because people copy&paste this from other packages without much thought ;-)

Reply:
Like I did :sweat_smile: I guess I will remove it then, no reason to stick to obsolete stuff